src/app/landing-page/landing-page.component.html

```diff
@@ -171,10 +171,11 @@ <h3 class="landing__prop-header">Everyone gets their own coin</h3>
     <div class="col-lg-8 col-md-9 col-sm-12 text-center landing__prop">
       <h3 class="landing__prop-header">Decentralized like Bitcoin</h3>
       <p class="landing__prop-desc">
+        This app is built on DeSo.
         <b>DeSo is not a company.</b>
         It's an open-source blockchain like Bitcoin.
         <br class="d-none d-md-block" />
-        You can own a piece by buying the DeSo coin $DESO.
+        You can own a piece by buying the $DESO coin.
         <br />
         <br />
         <a href="https://docs.deso.org/https://docs.deso.org/#the-ultimate-vision" target="_blank">Learn More</a>
```

Comment:
```suggestion
        <a href="https://docs.deso.org/#the-ultimate-vision" target="_blank">Learn More</a>
```
seems like a mistake I made when doing some renaming - sorry!